File: app/src/main/java/net/ddns/raspi_server/rezeptbuch/ui/SplashActivity.java
package net.ddns.raspi_server.rezeptbuch.ui;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;


public class SplashActivity extends AppCompatActivity {
  @Override
  protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);

    Intent intent = new Intent(this, MainActivity.class);
    startActivity(intent);
    finish();
  }
}

File: app/src/main/java/net/ddns/raspi_server/rezeptbuch/ui/SplashActivity.kt
package net.ddns.raspi_server.rezeptbuch.ui

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)

    val intent = Intent(this, MainActivity::class.java)
    startActivity(intent)
    finish()
  }
}
